packages/strapi-admin/package.json

```diff
@@ -56,7 +56,7 @@
     "hoist-non-react-statics": "^3.3.0",
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "^3.2.0",
-    "immer": "^7.0.14",
+    "immer": ">=8.0.1",
```

Comment:
you replace with an exact 8.0.1 or ^8.0.1 to avoid upgrading to major versions too